feature-examples/python/apikey_main.py

```diff
@@ -0,0 +1,86 @@
+"""Main file for Vectara feature examples related to ApiKey."""
+import argparse
+import logging
+import sys
+
+import create_apikey
+import delete_apikey
+import enable_apikey
+import list_apikeys
+import vectara_util
```

Comment:
Because of Python's import mechanism, I think it would be good to organize all Python example code under a `python` directory, with subdirectory names that use underscores rather than hyphens. So it might look like:
```
feature-examples
├── java
└── python
    ├── api_key
    │   ├── api_key_main.py
    │   ├── create_api_key.py
    │   ├── delete_api_key.py
    │   ├── enable_api_key.py
    │   └── list_api_key.py
    ├── feature_1
    ├── feature_2
    └── utils
        └── utils.py

```
All imports could then be absolute starting from the `python` directory.
This way, the user does not need to set up an extra PYTHONPATH to access the utility functions.

The result we are looking for is that the user can run something like `python3 api_key/api_key_main.py` from the `python` directory and have it work without PYTHONPATH manipulation.

Please use `api_key` instead of `apikey` in file names to be consistent with our docs.

Reply:
Done.